app/src/views/campaigns/settings.vue

```diff
@@ -1,10 +1,217 @@
 <template>
-  <div></div>
-</template>
+  <div class="action-container">
+    <el-form
+      ref="dataForm"
+      :model="tempData"
+      label-position="left"
+      label-width="200px"
+      status-icon
+      style="width: 100%; padding: 0 20px;"
+    >
+      <h1 style="padding-bottom: 20px;">{{ $t('route.campaigns.modify') }}</h1>
+
+      <h2>Description</h2>
+
+      <el-form-item
+        :label="$t('table.campaigns.name')"
+        prop="name"
+        :rules="[
+          {
+            required: true,
+            message: $t('form.campaigns.name.required'),
+            trigger: 'blur',
+          },
+        ]"
+      >
+        <el-input
+          v-model="tempData.name"
+          :placeholder="$t('campaigns.placeholder.name')"
+        />
+      </el-form-item>
+      <el-form-item
+        :label="$t('table.campaigns.description')"
+        prop="description"
+        :rules="[
+          {
+            required: true,
+            message: $t('form.campaigns.description.required'),
+            trigger: 'blur',
+          },
+        ]"
+      >
+        <el-input
+          type="textarea"
+          v-model="tempData.description"
+          :placeholder="$t('campaigns.placeholder.description')"
+        />
+      </el-form-item>
+      
+      <h2>Paramètres</h2>
+
+      <el-row :gutter="20">
+        <el-col :span="12">
+          <el-form-item
+            :label="$t('table.campaigns.isVisible')"
```

Comment:
Label : isVisible
Data: isPublish

Reply:
I wasn't sure that was what you meant. Is that right? 

```suggestion
            :label="$t('table.campaigns.isVisible')"
            :data="$t('table.campaigns.isPublish')"
```

Reply:
Nop, I mean you put a label call isVisible but the data link to this label is isPublish

Just keep the first line and set isPublish instead of isVisible

Reply:
```suggestion
            :label="$t('table.campaigns.isPublish')"
```